org-code-javabuilder/lib/src/test/java/org/code/javabuilder/JavabuilderTestExecutionListenerTest.java

```diff
@@ -70,9 +69,12 @@ public void testExecutionFinishedSendsStatusMessage() {
 
     final ClientMessage message = messageCaptor.getAllValues().get(0);
     assertEquals(ClientMessageType.TEST_RESULT, message.getType());
-    assertTrue(message.getValue().contains(displayName));
-    assertTrue(message.getValue().contains(classDisplayName));
-    assertTrue(message.getValue().contains(TestExecutionResult.Status.SUCCESSFUL.toString()));
+
+    JSONObject messageDetail = message.getDetail();
+    assertTrue(messageDetail.getString("methodName").equals(displayName));
```

Comment:
nit: use ClientMessageDetailKeys.METHOD_NAME, etc for these tests??

Reply:
good call